extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-validation.service.spec.ts

```diff
@@ -38,9 +38,10 @@ describe('DragAndDropSortInputValidationService', () => {
 
   let currentState: string;
```

Comment:
Just to check -- did you confirm that the tests in this file now fail when the "i = 0, j = 0, i = 0" bug is present?